sections/builder.js

```diff
@@ -1331,7 +1331,7 @@ export default [
     type: "method",
     method: "returning",
     example: ".returning(column) / .returning([column1, column2, ...])",
-    description: "Utilized by PostgreSQL, MSSQL, and Oracle databases, the returning method specifies which column should be returned by the insert and update methods. Passed column parameter may be a string or an array of strings. When passed in a string, makes the SQL result be reported as an array of values from the specified column. When passed in an array of strings, makes the SQL result be reported as an array of objects, each containing a single property for each of the specified columns. The returning method is not supported on Amazon Redshift.",
+    description: "Utilized by PostgreSQL, MSSQL, and Oracle databases, the returning method specifies which column should be returned by the insert and update methods. Passed column parameter may be a string or an array of strings. When passed in a string, makes the SQL result be reported as an array of values from the specified column. When passed in an array of strings, makes the SQL result be reported as an array of objects, each containing a single property for each of the specified columns. The returning method is not supported on Amazon Redshift or case of multiple inserts on MySQL and SQlite",
```

Comment:
I would suggest wrapping 'returning' in `\"`

`When passed in an array of strings` - not sure about syntax here, shouldn't it be "When an array of strings is passed in"?

`makes the SQL result be reported` -> "... to be reported"?

`The returning method is not supported on Amazon Redshift or case of multiple inserts on MySQL and SQlite"` -> "is entirely unsupported on Amazon Redshift and has no support for multiple inserts on MySQL and SQLite"?